M2/Macaulay2/packages/Book3264Examples.m2

```diff
@@ -64,7 +64,7 @@ placeholderToSchubertBasis(RingElement,FlagBundle) := (c,G) -> (
      (k,q) := toSequence(G.BundleRanks);
      P := diagrams(q,k);
      M := apply(P, i-> placeholderSchubertCycle(i,G));
-     E := flatten entries basis(R);
+     E := flatten entries basis(R, Variables => 0 .. numgens R - 1);
```

Comment:
This appears to depend on other changes to ``basis``, which I am personally not yet sold on.